AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/rootfragment/ActivityWithBeansWithRootFragment.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2016-2019 the AndroidAnnotations project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.test.rootfragment;
```

Comment:
Please move all this tests under `ebean` package. (`@RootContext` is also tested under that).

Reply:
That explains me why I didn't see any for @RootContext, moving them will required a little refactorization 😅 .